applications/allocation_result_mapper.py

```diff
@@ -45,10 +44,6 @@ def to_events(self):
                     basket_id=allocated_event.basket_id,
                 )
 
-                ApplicationEventStatus.objects.create(
```

Comment:
This left me pondering what state the application event will get when its gone through the allocation algorithm? Or does it change at all?

Reply:
I undid the delete. Now we se the event to `APPROVED` status